src/Microsoft.DotNet.Web.ProjectTemplates/content/RazorPagesWeb-CSharp/wwwroot/lib/jquery/.bower.json

```diff
@@ -12,14 +12,14 @@
     "library"
   ],
   "homepage": "https://github.com/jquery/jquery-dist",
-  "version": "2.2.0",
-  "_release": "2.2.0",
+  "version": "3.3.1",
+  "_release": "3.3.1",
   "_resolution": {
     "type": "version",
-    "tag": "2.2.0",
+    "tag": "3.3.1",
     "commit": "6fc01e29bdad0964f62ef56d01297039cdcadbe5"
   },
   "_source": "git://github.com/jquery/jquery-dist.git",
-  "_target": "2.2.0",
+  "_target": "3.3.1",
```

Comment:
Likewise, did you modify the version number by hand, or is this from an updated package? If it was done by hand, we should probably have the `commit` hashes consistent with the `tag`.